Source/Hurl.BrowserSelector/Controls/BrowserButton.xaml.cs

```diff
@@ -13,8 +13,11 @@ public partial class BrowserButton : UserControl
     {
         public BrowserButton(Browser browser)
         {
-            InitializeComponent();
-            DataContext = browser;
+            if (browser.Hidden == false)
```

Comment:
While this might fix the issue, A better solution would be to fix it during the query to list of browsers as seen below

https://github.com/U-C-S/Hurl/blob/a0dedeb646ff1f1a37f41a6f6a84ca09e4840358/Source/Hurl.BrowserSelector/Windows/MainWindow.xaml.cs#L73-L80

Reply:
I placed the `if (!browser.Hidden)` into `MainWindow.xaml.cs` instead of `BrowserButton.xaml.cs` [Commits here](https://github.com/U-C-S/Hurl/pull/164/commits/4171b0ef1e5785d29f0d9db57312ed685516c3d0)

It works just as expected.